x-pack/test/cloud_security_posture_functional/page_objects/csp_dashboard_page.ts

```diff
@@ -63,6 +64,7 @@ export function CspDashboardPageProvider({ getService, getPageObjects }: FtrProv
 
     getKubernetesTab: async () => {
       const tabs = await dashboard.getDashboardTabs();
+      sleep(1000);
```

Comment:
Instead of placing a sleep here we should check if we can wait for the tabs to appear with a timeout of 1s or 2s